build.zig

```diff
@@ -18,25 +18,27 @@ pub fn build(b: *std.build.Builder) void {
     };
     const options = Options{ .gpu_dawn_options = gpu_dawn_options };
 
-    const main_tests = b.addTest("src/main.zig");
+    const main_tests = b.addTestExe("mach-tests", "src/main.zig");
     main_tests.setBuildMode(mode);
     main_tests.setTarget(target);
     main_tests.addPackage(pkg);
     main_tests.addPackage(gpu.pkg);
     main_tests.addPackage(glfw.pkg);
+    main_tests.install();
```

Comment:
Is it actually right to _install_ tests? This would place them in the `zig-out` directory as I understand it, which feels not ideal to me. For example, if we were to begin publishing binary releases I don't think we would want these included.

Reply:
i think we should just wait for zig to provide a way to compile without producing binaries